robotpy_build/platforms.py

```diff
@@ -51,6 +50,12 @@ def get_platform() -> WPILibMavenPlatform:
     #       be useful to note for the future.
 
     pyplatform = _get_platform()
+
+    # Check for 64 bit macOS (version agnostic)
+    if 'osx' in pyplatform and '64' in pyplatform:
```

Comment:
I don't like this. This will pick up ye olde PPC64.

Reply:
Good catch, Now uses pattern matching. It should support osx 10.4 - current.